pandas/_libs/src/headers/stdint.h

```diff
@@ -1,10 +0,0 @@
-#ifndef _PANDAS_STDINT_H_
-#define _PANDAS_STDINT_H_
-
-#if defined(_MSC_VER) && (_MSC_VER < 1900)
```

Comment:
I guess the shim was only used for < VS 2015(https://dev.to/yumetodo/list-of-mscver-and-mscfullver-8nd).  We should be good here as our min. Visual Studio requirement should be VS2019.

Reply:
More of out curiosity than anything else, where do we specify the minimum version for VS?

Reply:
Absolutely no idea I opened cython/cython#2570 years ago in the hope this would become their problem

Reply:
> More of out curiosity than anything else, where do we specify the minimum version for VS?


There's a mention of VS2019 in our development environment docs

https://pandas.pydata.org/pandas-docs/stable/development/contributing_environment.html#installing-a-c-compiler